src/app/services/evnironment-variables.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from "@angular/core";
+import { EnvironmentVariablesApi } from "../api/environment-variables.api";
+import { map, Observable } from "rxjs";
+import {
+    DeleteEnvVariableMutation,
+    ExposedEnvVariableValueQuery,
+    ListEnvVariablesQuery,
+    ModifyEnvVariableMutation,
+    SaveEnvVariableMutation,
+    ViewDatasetEnvVarConnection,
+} from "../api/kamu.graphql.interface";
+import { ToastrService } from "ngx-toastr";
+
+@Injectable({
+    providedIn: "root",
+})
+export class EvnironmentVariablesService {
```

Comment:
This feature is specific to datasets, but you've named it as if we were talking about some global env vars. Use word "dataset" in the service name or in methods names.

Reply:
Renamed